draft-ietf-quic-transport.md

```diff
@@ -486,9 +486,22 @@ Key Phase Bit:
   recipient of a packet to identify the packet protection keys that are used to
   protect the packet.  See {{QUIC-TLS}} for details.
 
+\[\[RFC editor: please remove the Google QUIC Demultipexing bit before
+publication.]]
+
+Google QUIC Demultipexing Bit:
+
+: The fifth bit (0x8) of octet 0 is set to 0. This allows implementations of
+  Google QUIC to distinguish Google QUIC packets from short header packets sent
+  by a client because Google QUIC servers expect the connection ID to always be
+  present.
+  The special interpretation of this bit SHOULD be removed from this
+  specification when Google QUIC has finished transitioning to the new header
+  format.
+
```

Comment:
Can you add a paragraph for reserving the 4th bit as well? Maybe call it the STUN Demultiplexing Bit?